glide.lock

```diff
@@ -166,7 +166,7 @@ imports:
   - pkg/utils/hwaddr
 - name: github.com/containers/image
   version: 9dc3776eaea701cad0c09f04ec489c35a520e2b2
-  repo: https://github.com/openshift/containers-image.git
+  repo: https://github.com/openshift/containers-image
```

Comment:
These changes in glide.lock will go away - I need to merge in API changes, which will include some refactoring.